core/controllers/admin_test.py

```diff
@@ -2446,9 +2446,10 @@ def setUp(self):
         super(DeleteUserHandlerTest, self).setUp()
         self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
         self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
-        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
-        self.login(self.ADMIN_EMAIL, is_super_admin=True)
-        self.admin_user_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.signup(feconf.SYSTEM_EMAIL_ADDRESS, self.ADMIN_USERNAME)
```

Comment:
Just checking: is ``SYSTEM_EMAIL_ADDRESS`` actually defined in this test environment or is it empty string?

Reply:
It is defined to some random non-existent email address.